osu.Desktop/Program.cs

```diff
@@ -25,17 +26,22 @@ public static class Program
         public static int Main(string[] args)
         {
             LegacyFilesystemReader.Register();
+            
+            var cwd = Directory.GetCurrentDirectory();
 
             using (DesktopGameHost host = Host.GetSuitableHost(@"osu", true))
             {
                 if (!host.IsPrimaryInstance)
                 {
                     var importer = new BeatmapImporter(host);
-
+                    Directory.SetCurrentDirectory(cwd);
                     foreach (var file in args)
-                        if (!importer.Import(file).Wait(1000))
+                    {
+                        Console.WriteLine(@"Importing {0}", file);
+                        if (!importer.Import(Path.GetFullPath(file)).Wait(3000))
                             throw new TimeoutException(@"IPC took too long to send");
-                    Console.WriteLine(@"Sent import requests to running instance");
+                        Thread.Sleep(500);

```

Comment:
Arbitrary number to slow it down, since the server doesn't tell us when it's done processing an import. We could refactor this but should be done separately imo.